requirements.txt

```diff
@@ -16,7 +16,7 @@ cffi==1.15.0
 charset-normalizer==2.0.9
 click==8.0.3
 cryptography==36.0.0
-funcx-common==0.0.10
+funcx-common[redis,boto3]==0.0.11
```

Comment:
This is not what's output by `make freezedeps`. Did you make this change manually? `requirements.txt` has a comment at the top stating that it should be created via `make freezedeps`.

(I'd like to move to poetry to codify this process with a dedicated tool, but there are enough other things in motion that it's hard to get that done.)

Reply:
I tried making the file with freezedeps, but there were a ton of changes that I wasn't confident with. I can commit that file if that's better.

Reply:
`requirements.in` states the constraints which we're applying to get our requirements data. `requirements.txt` is a freeze being used to record the versions which are deployed.

If you manually update `requirements.txt`, you aren't guaranteed to be producing something which works at install time, and we should generally not do that.

In the future, when we have a tool doing this rather than something hacked up to get things stable, this will be codified even more strongly. e.g. Under `poetry`, nobody manually writes the `poetry.lock` file -- ownership of that file belongs to the tool.

I'm very strongly against manually tinkering with this file.

Reply:
Done.